apps/ledger-live-mobile/src/components/AccountSelector.tsx

```diff
@@ -28,6 +30,14 @@ const AccountSelector = ({
 }: Props) => {
   const { t } = useTranslation();
   const accounts = useSelector(accountsSelector);
+  const walletState = useSelector(walletSelector);
+
+  const items = useMemo(() => {
+    return list.map(account => ({
+      account,
+      name: accountNameWithDefaultSelector(walletState, account),
```

Comment:
yes, it's used by `FilteredSearchBar` and notably with the SEARCH_KEYS that have a "name". 
i had to change this code specifically for this complex search engine system, since the .name was no longer in account, i had to make a higher level object.

Reply:
I see, too bad we have to map again in the `renderList`